components/server/src/test/java/com/hotels/styx/server/netty/codec/NettyToStyxRequestDecoderTest.java

```diff
@@ -332,7 +333,9 @@ private FullHttpResponse send(HttpRequest request) {
     }
 
     private void assertThatHttpHeadersAreSame(Iterable<HttpHeader> headers, HttpHeaders headers1) {
-        assertThat(newArrayList(headers).toString(), is(newArrayList(headers1).toString()));
+        for (HttpHeader header : headers) {
```

Comment:
This doesn't verify that headers1 is the same size as headers.

Reply:
fixed